tests/DateServiceTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+use PHPUnit\Framework\TestCase;
+
+class DateServiceTest extends TestCase
+{
+    public function testConvertToToUkFormat()
```

Comment:
this test should say the word success within it as it is a success test (like how you've done for malformed and failure)

probably worth writing on WB how we all should be writing test names